go/kv/internal.go

```diff
@@ -65,3 +65,14 @@ func (this *internalKVStore) GetKeyValue(key string) (value string, found bool,
 func (this *internalKVStore) DistributePairs(kvPairs [](*KVPair)) (err error) {
 	return nil
 }
+
+func (this *internalKVStore) DeleteRecursive(key string) (err error) {
+	_, err = db.ExecOrchestrator(`
+		delete
+			from kv_store 
+		where
+		store_key rlike ?
```

Comment:
will this delete `/mysql/master/important` when we forget the `import` cluster, seeing that `import` is a prefix of `important`?

Reply:
That's true, my mistake, sorry. Replace it to like

Reply:
Now it's even worse 😂 .